internal/json/json.go

```diff
@@ -2,8 +2,10 @@
 // Use of this source code is governed by a MIT style
 // license that can be found in the LICENSE file.
 
-//go:build !jsoniter && !go_json
-// +build !jsoniter,!go_json
+//go:build !jsoniter && !go_json && !(sonic && avx && (linux || windows || darwin) && amd64 && (go1.15 || go1.16 || go1.17 || go1.18))
```

Comment:
why set go version?

Reply:
@thinkerou as describe by @Bisstocuz, I add the all limits of sonic with build tags, if you think this is no use, I'd like to delete them.

Reply:
because `go: [1.15, 1.16, 1.17, 1.18]` has limited, so we should remove it.